db/migrate/20210728100307_fix_reporting_months_start_date.rb

```diff
@@ -0,0 +1,6 @@
+class FixReportingMonthsStartDate < ActiveRecord::Migration[5.2]
+  def up
+    execute "SET LOCAL TIME ZONE '#{Rails.application.config.country[:time_zone]}'"
+    replace_view :reporting_months, version: 3, revert_to_version: 2, materialized: false
```

Comment:
We can use `replace_view` here since it's not a matview, and postgres lets us do this as long as the view's schema remains intact.